scripts/staging-ci/merge-into-staging.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+const { initGitConfig, executeCommand, printLog, printError } = require('../utils')
+
+const REPOSITORY = process.env.GIT_REPOSITORY
+const CURRENT_STAGING_BRANCH = process.env.CURRENT_STAGING
+const CI_COMMIT_SHA = process.env.CI_COMMIT_SHA
+const CI_COMMIT_REF_NAME = process.env.CI_COMMIT_REF_NAME
+
+async function main() {
+  await initGitConfig(REPOSITORY)
+  await executeCommand(`git fetch --no-tags origin ${CURRENT_STAGING_BRANCH}`)
+  await executeCommand(`git checkout ${CURRENT_STAGING_BRANCH} -f`)
+  await executeCommand(`git pull origin ${CURRENT_STAGING_BRANCH}`)
+
+  printLog(`Merging ${CI_COMMIT_REF_NAME} (${CI_COMMIT_SHA}) with ${CURRENT_STAGING_BRANCH}...`)
+  try {
+    await executeCommand(`git merge --no-ff "${CI_COMMIT_SHA}"`)
+  } catch {
+    const diff = await executeCommand(`git diff`)
```

Comment:
Could you print the caught error too? To be aware if the merge fails with anything other than a conflict we.

Reply:
Sure